CanalTP/Chaos
# Update filter impacts for line_section

# Description

This PR add the return of line_section objects from URI /impacts in order to show them on the calendar part in NMM

## Issue

Issue link: BOT-134

## How to test

Here are the following steps to test this pull request:

- Create an impact with a line_section objects
- Call chaos API /impacts with parameters : uri[]={line_uri_of_line_section}
- You should retrieve the impact
